src/client/datascience/jupyter/liveshare/hostJupyterExecution.ts

```diff
@@ -75,7 +75,7 @@ export class HostJupyterExecution extends JupyterExecutionBase implements IJupyt
     }
 
     public async hostConnectToNotebookServer(
-        options?: INotebookServerOptions,
+        options: INotebookServerOptions,

```

Comment:
This can never be null, hence much easier now, 
This makes a big difference as the property disableUI is being passed around and i need to know the value, but if we have optionals all over the place then we need to be careful, this helps with that.

Looks like we alway pass values, except in tests (which is weird, as we're testing a scenario thats not really in the realworld)